docs/socketexample.rst

```diff
@@ -552,7 +569,7 @@ need this method.
 .. doctest::
 
    >>> utils = list(zope.component.getAllUtilitiesRegisteredFor(IUSSocket))
-   >>> utils.sort()
+   >>> utils.sort(key=lambda x: type(x).__name__)
```

Comment:
That starts to obscure what's going on, IMO. I'll just use `sorted` and deal with the indentation.